sorting/MergeSort.r

```diff
@@ -0,0 +1,31 @@
+ merge.merge <- function(a, b) {
+   r <- numeric(length(a)+length(b))
+   ai <- 1
+   bi <- 1
+   j<-1
+   for(j in 1:length(r)) {
+     if((ai<=length(a) && a[ai]<b[bi]) || bi>length(b)) {
```

Comment:
Use proper spacing before and after all sorts of operators. (In your code, there is none between the relational ones, while there is a gap for the logical ones)